### ISSUE TITLE
Live TV M3U not working.

### ISSUE THREAD

**Issue by gitdeath** [jellyfin/jellyfin#4783]
**Describe the bug**
Live TV tuner from M3U does not play

**System (please complete the following information):**
 - OS: Synology
 - Virtualization: Docker
 - Clients: Chrome Desktop Browser, Pixel 3a Android, Fire Kids Tablet, Pixel 3 Android
 - Browser: Chrome Version 87.0.4280.88 (Official Build) (64-bit)
 - Jellyfin Version: 10.6.4
 - Playback: Direct Play, Remux, Direct Stream, Transcode  
 - Installed Plugins: Kodi Sync
 - Reverse Proxy: nginx - I get the same issue if I go directly to the internal IP, thus bypassing the reverse proxy issue.
 - Base URL: none
 - Networking: Host
 - Storage: n/a for Live TV

**To Reproduce**

Attempt to start TV from Guide, Schedule, Channels, Programs, or On Now

**Expected behavior**
Live TV channel to start

**Logs**
```
[2020-12-13 15:23:10.668 -06:00] [INF] [32] MediaBrowser.Api.Playback.MediaInfoService: User policy for "<username removed>". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2020-12-13 15:23:10.669 -06:00] [INF] [32] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2020-12-13 15:23:10.669 -06:00] [INF] [32] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", No direct play profiles found for Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:23:10.669 -06:00] [INF] [32] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2020-12-13 15:23:10.670 -06:00] [INF] [32] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", No direct play profiles found for Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:23:10.671 -06:00] [INF] [32] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "m3u_677f2cb9fd3aab353c91efdde919f522171ca7311c1651375649905063cb7d3a"
[2020-12-13 15:23:10.671 -06:00] [INF] [32] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel m3u_677f2cb9fd3aab353c91efdde919f522171ca7311c1651375649905063cb7d3a
[2020-12-13 15:23:10.672 -06:00] [INF] [32] Emby.Server.Implementations.LiveTv.TunerHosts.M3UTunerHost: Opening SharedHttpStream Live stream from "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:23:10.992 -06:00] [ERR] [57] Emby.Server.Implementations.LiveTv.TunerHosts.M3UTunerHost: Error opening tuner
MediaBrowser.Model.Net.HttpException: Found
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.LiveTv.TunerHosts.SharedHttpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2020-12-13 15:23:10.994 -06:00] [ERR] [57] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: "http://jellyfin.<domain removed>/Items/058810376b43e25159848a10f4d151e5/PlaybackInfo?UserId=a44692fa1add4d22af9e77add31b2f24&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStream(LiveStreamRequest request, CancellationToken cancellationToken)
   at MediaBrowser.Api.Playback.MediaInfoService.OpenMediaSource(OpenMediaSource request)
   at MediaBrowser.Api.Playback.MediaInfoService.GetPlaybackInfo(GetPostedPlaybackInfo request)
   at MediaBrowser.Api.Playback.MediaInfoService.Post(GetPostedPlaybackInfo request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
[2020-12-13 15:24:20.425 -06:00] [WRN] [26] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to "10.0.0.5". Time (slow): 0:00:00.7212512. "http://jellyfin.<domain removed>/Users/a44692fa1add4d22af9e77add31b2f24/Items?SortBy=SortName&SortOrder=Ascending&IncludeItemTypes=Series&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&StartIndex=0&Limit=100&ParentId=767bffe4f11c93ef34b805451a696a4e"
[2020-12-13 15:24:25.873 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: GetPostedPlaybackInfo profile: DeviceProfile { Name: null, Id: null, Identification: null, FriendlyName: null, Manufacturer: null, ManufacturerUrl: null, ModelName: null, ModelDescription: null, ModelNumber: null, ModelUrl: null, SerialNumber: null, EnableAlbumArtInDidl: False, EnableSingleAlbumArtLimit: False, EnableSingleSubtitleLimit: False, SupportedMediaTypes: "Audio,Photo,Video", UserId: null, AlbumArtPn: null, MaxAlbumArtWidth: 0, MaxAlbumArtHeight: 0, MaxIconWidth: null, MaxIconHeight: null, MaxStreamingBitrate: 120000000, MaxStaticBitrate: 100000000, MusicStreamingTranscodingBitrate: 192000, MaxStaticMusicBitrate: null, SonyAggregationFlags: null, ProtocolInfo: null, TimelineOffsetSeconds: 0, RequiresPlainVideoItems: False, RequiresPlainFolders: False, EnableMSMediaReceiverRegistrar: False, IgnoreTranscodeByteRangeRequests: False, XmlRootAttributes: [], DirectPlayProfiles: [DirectPlayProfile { Container: "webm", AudioCodec: "vorbis,opus", VideoCodec: "vp8,vp9,av1", Type: Video }, DirectPlayProfile { Container: "mp4,m4v", AudioCodec: "mp3,aac,opus,flac,vorbis", VideoCodec: "h264,vp8,vp9,av1", Type: Video }, DirectPlayProfile { Container: "mov", AudioCodec: "mp3,aac,opus,flac,vorbis", VideoCodec: "h264", Type: Video }, DirectPlayProfile { Container: "opus", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "mp3", AudioCodec: "mp3", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "aac", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "m4a,m4b", AudioCodec: "aac", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "flac", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "webma,webm", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "wav", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "ogg", AudioCodec: null, VideoCodec: null, Type: Audio }], TranscodingProfiles: [TranscodingProfile { Container: "ts", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "hls", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 1, SegmentLength: 0, BreakOnNonKeyFrames: True }, TranscodingProfile { Container: "aac", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "opus", Type: Audio, VideoCodec: null, AudioCodec: "opus", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "wav", Type: Audio, VideoCodec: null, AudioCodec: "wav", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "opus", Type: Audio, VideoCodec: null, AudioCodec: "opus", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "aac", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "wav", Type: Audio, VideoCodec: null, AudioCodec: "wav", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "ts", Type: Video, VideoCodec: "h264", AudioCodec: "mp3,aac", Protocol: "hls", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 1, SegmentLength: 0, BreakOnNonKeyFrames: True }, TranscodingProfile { Container: "webm", Type: Video, VideoCodec: "vpx", AudioCodec: "vorbis", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "2", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp4", Type: Video, VideoCodec: "h264", AudioCodec: "mp3,aac,opus,flac,vorbis", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }], ContainerProfiles: [], CodecProfiles: [CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: Equals, Property: IsSecondaryAudio, Value: "false", IsRequired: False }], ApplyConditions: [], Codec: "aac", Container: null }, CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: Equals, Property: IsSecondaryAudio, Value: "false", IsRequired: False }], ApplyConditions: [], Codec: null, Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: NotEquals, Property: IsAnamorphic, Value: "true", IsRequired: False }, ProfileCondition { Condition: EqualsAny, Property: VideoProfile, Value: "high|main|baseline|constrained baseline|high 10", IsRequired: False }, ProfileCondition { Condition: LessThanEqual, Property: VideoLevel, Value: "51", IsRequired: False }, ProfileCondition { Condition: NotEquals, Property: IsInterlaced, Value: "true", IsRequired: False }], ApplyConditions: [], Codec: "h264", Container: null }], ResponseProfiles: [ResponseProfile { Container: "m4v", AudioCodec: null, VideoCodec: null, Type: Video, OrgPn: null, MimeType: "video/mp4", Conditions: [] }], SubtitleProfiles: [SubtitleProfile { Format: "vtt", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: External, DidlMode: null, Language: null, Container: null }] }
[2020-12-13 15:24:25.922 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: User policy for "<username removed>". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2020-12-13 15:24:25.922 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2020-12-13 15:24:25.922 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", No direct play profiles found for Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:24:25.923 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2020-12-13 15:24:25.923 -06:00] [INF] [74] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", No direct play profiles found for Path: "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:24:25.924 -06:00] [INF] [74] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "m3u_677f2cb9fd3aab353c91efdde919f522171ca7311c1651375649905063cb7d3a"
[2020-12-13 15:24:25.924 -06:00] [INF] [74] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel m3u_677f2cb9fd3aab353c91efdde919f522171ca7311c1651375649905063cb7d3a
[2020-12-13 15:24:25.925 -06:00] [INF] [74] Emby.Server.Implementations.LiveTv.TunerHosts.M3UTunerHost: Opening SharedHttpStream Live stream from "https://cdn.sha-256.org/1011571027/ArnSTcXnMP/286"
[2020-12-13 15:24:26.533 -06:00] [ERR] [57] Emby.Server.Implementations.LiveTv.TunerHosts.M3UTunerHost: Error opening tuner
MediaBrowser.Model.Net.HttpException: Found
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.LiveTv.TunerHosts.SharedHttpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2020-12-13 15:24:26.536 -06:00] [ERR] [57] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: "http://jellyfin.<domain removed>/Items/058810376b43e25159848a10f4d151e5/PlaybackInfo?UserId=a44692fa1add4d22af9e77add31b2f24&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStream(LiveStreamRequest request, CancellationToken cancellationToken)
   at MediaBrowser.Api.Playback.MediaInfoService.OpenMediaSource(OpenMediaSource request)
   at MediaBrowser.Api.Playback.MediaInfoService.GetPlaybackInfo(GetPostedPlaybackInfo request)
   at MediaBrowser.Api.Playback.MediaInfoService.Post(GetPostedPlaybackInfo request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
[2020-12-13 15:24:26.539 -06:00] [WRN] [57] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 500 to "10.0.0.5". Time (slow): 0:00:00.6820965. "http://jellyfin.<domain removed>/Items/058810376b43e25159848a10f4d151e5/PlaybackInfo?UserId=a44692fa1add4d22af9e77add31b2f24&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000"
[2020-12-13 15:25:58.933 -06:00] [INF] [63] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:192.168.1.201" closed
[2020-12-13 15:26:00.273 -06:00] [WRN] [67] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to "10.0.0.5". Time (slow): 0:00:00.5825185. "http://jellyfin.<domain removed>/System/Info"
[2020-12-13 15:26:00.290 -06:00] [INF] [67] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:192.168.1.201" request
[2020-12-13 15:26:01.238 -06:00] [WRN] [57] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to "10.0.0.5". Time (slow): 0:00:00.727566. "http://jellyfin.<domain removed>/Items/e7929b7948330fcb6f2e1deec9d4baae/Images/Backdrop/0?tag=ce055f0bd01ad0cd907aca3e4e9dc581&maxWidth=1920&quality=80"
[2020-12-13 15:26:49.254 -06:00] [WRN] [41] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to "10.0.0.5". Time (slow): 0:00:00.7352365. "http://jellyfin.<domain removed>/Items/80afeabe36ba7d7aca4f13326be88997/Images/Backdrop/0?tag=5428e8d7872c4a0cf67d6a6cd874e3a2&maxWidth=1920&quality=80"
[2020-12-13 15:27:13.260 -06:00] [WRN] [41] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to "10.0.0.5". Time (slow): 0:00:00.7346953. "http://jellyfin.<domain removed>/Items/37ff54010316c51d34763598ac334279/Images/Backdrop/0?tag=9f8997d1bf200f480b35beff9e51cdb5&maxWidth=1920&quality=80"
```

**Screenshots**

![image](https://user-images.githubusercontent.com/59674699/102024702-337ac680-3d59-11eb-8c46-493a1976c48f.png)

**Additional context**

These streams work from Kodi directly. I recently switched form the jellyfin/linuxserver image to the jellyfin/jellyfin image (I did a completely fresh build, didn't reuse databases or anything.) I had this setup working in the linuxserver version.

**Comment by crobibero** [comment]
I believe we have already fixed this for 10.7

**Comment by gitdeath** [comment]
I just pulled the 10.7.0 RC1 image and it is working.
